en/debug/system_console.md

```diff
@@ -2,18 +2,21 @@
 
 The system console allows low-level access to the system, debug output and analysis of the system boot process. The most convenient way to connect it is by using a [Dronecode probe](http://nicadrone.com/index.php?id_product=61&controller=product), but a plain FTDI cable can be used as well.
 
-## System Console vs. Shell
+## System Console vs. Shells
 
-There are multiple shells, but only one console: The system console is the location where all boot output (and applications auto-started on boot) is printed.
+There is just one *System Console*, which runs on one specific UART (the debug port, as configured in NuttX), and is commonly attached via FTDI cable.
+- Used for *low-level debugging/development*: bootup, NuttX, startup scripts, board bringup, development on central parts of PX4 (e.g. uORB).
+- In particular, is the only place where all boot output (including information about applications auto-started on boot) is printed.
 
-  * System console (first shell): Hardware serial port
-  * Additional shells: Pixhawk on USB (e.g. lists as /dev/tty.usbmodem1 on Mac OS)
+Shells provide higher-level access to the system:
+- Used for basic module testing/running commands.
+- Only display the output of modules you start (and therefore cannot debug the boot process).
+- Cannot display the output of tasks running on the work queue.
+
+There are several shells:
+* USB shell (e.g. Pixhawk on USB lists as `/dev/tty.usbmodem1` on Mac OS)
```

Comment:
This also runs mavlink now. We can write that the shell can be used on any UART (including USB), but running mavlink is more powerful and thus we only use the shell via mavlink today.

Reply:
Can you update appropriately? What is wrong with this (as it is now) is that is isn't "really" clear how you would use it - ie I know how to connect up to use with QGC, but how do I use the USB option?

Reply:
Done